debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -461,6 +461,14 @@ public static SchemaRefreshMode parse(String value) {
                                                                 "Whether or not to drop the logical replication slot when the connector finishes orderly" +
                                                                 "By default the replication is kept so that on restart progress can resume from the last recorded location");
 
+    public static final Field STREAM_PARAMS = Field.create("slot.stream_params")
```

Comment:
Would it be possible to use dot insted of underscore?

Reply:
Done.